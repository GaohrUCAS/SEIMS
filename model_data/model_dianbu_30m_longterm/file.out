### FORMAT: TAG | VALUE
### ALL AVAILABLE TAGS:
### OUTPUTID	: VARIABLE NAME FOR OUTPUT
### COUNT		: OUTPUT NUMBER, e.g., 2 WHEN OUTPUT PET WITH AGGREGATION TYPE OF SUM AND MEAN.
### TYPE		: DATA AGGREGATION TYPE, e.g., SUM, AVE, MAX, etc.
### STARTTIME	: DATA DERIVED TIME START
### ENDTIME		: DATA DERIVED TIME END
### FILENAME	: OUTPUT FILENAME WITH SUFFIX, e.g., txt, asc, tif.

OUTPUTID|D_P
COUNT | 1
TYPE | SUM
STARTTIME|2014-01-01 00:00:00
ENDTIME|2014-12-31 00:00:00
FILENAME | PREC_SUM.tif

OUTPUTID|PET
COUNT | 2
TYPE | SUM
STARTTIME|2014-01-01 00:00:00
ENDTIME|2014-12-31 00:00:00
FILENAME | PET_SUM.tif
TYPE | MEAN
STARTTIME|2014-01-01 00:00:00
ENDTIME|2014-12-31 00:00:00
FILENAME | PET_MEAN.tif

# OUTPUTID|NEPR
# COUNT | 1
# TYPE | SUM
# STARTTIME|2014-01-01 00:00:00
# ENDTIME|2014-12-31 00:00:00
# FILENAME|NEPR_1

# OUTPUTID|SOTE
# COUNT | 1
# TYPE | AVERAGE
# STARTTIME|2014-01-01 00:00:00
# ENDTIME|2014-12-31 00:00:00
# FILENAME|SOTE_1

# OUTPUTID|INFIL
# COUNT | 1
# TYPE | SUM
# STARTTIME|2014-01-01 00:00:00
# ENDTIME|2014-12-31 00:00:00
# FILENAME|INFIL_1

# OUTPUTID|EXCP
# COUNT | 1
# TYPE | SUM
# STARTTIME|2014-01-01 00:00:00
# ENDTIME|2014-12-31 00:00:00
# FILENAME|EXCP_1

# OUTPUTID|SURU
# COUNT | 1
# TYPE | SUM
# STARTTIME|2014-01-01 00:00:00
# ENDTIME|2014-12-31 00:00:00
# FILENAME|SURU_1

# OUTPUTID|Percolation_2D
# COUNT | 1
# TYPE | SUM
# STARTTIME|2014-01-01 00:00:00
# ENDTIME|2014-12-31 00:00:00
# FILENAME|Percolation_2D

# OUTPUTID|SSRU_2D
# COUNT | 1
# TYPE | SUM
# STARTTIME|2014-01-01 00:00:00
# ENDTIME|2014-12-31 00:00:00
# FILENAME|SSRU_2D

# OUTPUTID|SOER
# COUNT | 1
# TYPE | SUM
# STARTTIME|2014-01-01 00:00:00
# ENDTIME|2014-12-31 00:00:00
# FILENAME|SOER

# OUTPUTID|INET
# COUNT | 1
# TYPE | SUM
# STARTTIME|2014-01-01 00:00:00
# ENDTIME|2014-12-31 00:00:00
# FILENAME|INET_1

# # OUTPUTID|SNME
# # COUNT | 1
# # TYPE | SUM
# # STARTTIME|2014-01-01 00:00:00
# # ENDTIME|2014-12-31 00:00:00
# # FILENAME|SNME_1

# # OUTPUTID|SNAC
# # COUNT | 1
# # TYPE | SUM
# # STARTTIME|2014-01-01 00:00:00
# # ENDTIME|2014-12-31 00:00:00
# # FILENAME|SNAC_1

# # OUTPUTID | Nitrate
# # COUNT | 1
# # TYPE | SUM
# # STARTTIME|2014-01-01 00:00:00
# # ENDTIME|2014-12-31 00:00:00
# # FILENAME | Nitrate

# # OUTPUTID | Ammon
# # COUNT | 1
# # TYPE | SUM
# # STARTTIME|2014-01-01 00:00:00
# # ENDTIME|2014-12-31 00:00:00
# # FILENAME | Ammon

# # OUTPUTID | LANITTOCH
# # SUBBASINCOUNT | 1
# # SUBBASINID | 0
# # STARTTIME|2014-01-01 00:00:00
# # ENDTIME|2014-12-31 00:00:00
# # FILENAME | LANITTOCH.txt

# # OUTPUTID | SNITTOCH_T
# # SUBBASINCOUNT | 1
# # SUBBASINID | 0
# # STARTTIME|2014-01-01 00:00:00
# # ENDTIME|2014-12-31 00:00:00
# # FILENAME | SNITTOCH_T.txt

# # OUTPUTID | SSOLPTOCH_T
# # SUBBASINCOUNT | 1
# # SUBBASINID | 0
# # STARTTIME|2014-01-01 00:00:00
# # ENDTIME|2014-12-31 00:00:00
# # FILENAME | SSOLPTOCH_T.txt

# # OUTPUTID | ATTPTOCH
# # SUBBASINCOUNT | 1
# # SUBBASINID | 0
# # STARTTIME|2014-01-01 00:00:00
# # ENDTIME|2014-12-31 00:00:00
# # FILENAME | ATTPTOCH.txt

# # OUTPUTID | NITOUTLET
# # SUBBASINCOUNT | 1
# # SUBBASINID | 0
# # STARTTIME|2014-01-01 00:00:00
# # ENDTIME|2014-12-31 00:00:00
# # FILENAME | NITOUTLET.txt

# # OUTPUTID | AMMOUTLET
# # SUBBASINCOUNT | 1
# # SUBBASINID | 0
# # STARTTIME|2014-01-01 00:00:00
# # ENDTIME|2014-12-31 00:00:00
# # FILENAME | AMMOUTLET.txt

# OUTPUTID | SEDTOCH_T
# SUBBASINCOUNT | 1
# SUBBASINID | 0
# STARTTIME|2014-01-01 00:00:00
# ENDTIME|2014-12-31 00:00:00
# FILENAME | SEDTOCH_T.txt

# OUTPUTID | SBOF
# SUBBASINCOUNT | 1
# SUBBASINID | 0
# STARTTIME|2014-01-01 00:00:00
# ENDTIME|2014-12-31 00:00:00
# FILENAME | SBOF.txt

# OUTPUTID | SBQG
# SUBBASINCOUNT | 1
# SUBBASINID | 0
# STARTTIME|2014-01-01 00:00:00
# ENDTIME|2014-12-31 00:00:00
# FILENAME | SBQG.txt

#OUTPUTID | QRECH
#SUBBASINCOUNT | 1
#SUBBASINID | 0
#STARTTIME|2014-01-01 00:00:00
#ENDTIME|2014-12-31 00:00:00
#FILENAME | Q.txt
