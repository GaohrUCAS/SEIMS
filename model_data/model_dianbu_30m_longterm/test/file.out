### FORMAT: TAG | VALUE
### ALL AVAILABLE TAGS:
### OUTPUTID	: VARIABLE NAME FOR OUTPUT
### COUNT		: OUTPUT NUMBER, e.g., 2 WHEN OUTPUT PET WITH AGGREGATION TYPE OF SUM AND MEAN.
### TYPE		: DATA AGGREGATION TYPE, e.g., SUM, AVE, MAX, etc.
### STARTTIME	: DATA DERIVED TIME START
### ENDTIME		: DATA DERIVED TIME END
### FILENAME	: OUTPUT FILENAME WITH SUFFIX, e.g., txt, asc, tif.

### -1 Module IO TEST
OUTPUTID|CN2_M
COUNT | 1
TYPE | MEAN
STARTTIME|2014-01-01 00:00:00
ENDTIME|2014-12-31 00:00:00
FILENAME | CN2_M.tif

OUTPUTID|K_M
COUNT | 1
TYPE | MAX
STARTTIME|2014-01-01 00:00:00
ENDTIME|2014-12-31 00:00:00
FILENAME | K_M.tif
